homeassistant/components/flick_electric/sensor.py

```diff
@@ -36,20 +34,16 @@ class FlickPricingSensor(SensorEntity):
 
     _attr_attribution = "Data provided by Flick Electric"
     _attr_native_unit_of_measurement = f"{CURRENCY_CENT}/{UnitOfEnergy.KILO_WATT_HOUR}"
+    _attr_translation_key = "power_price"
 
     def __init__(self, api: FlickAPI) -> None:
         """Entity object for Flick Electric sensor."""
         self._api: FlickAPI = api
         self._price: FlickPrice = None
         self._attributes: dict[str, Any] = {
-            ATTR_FRIENDLY_NAME: FRIENDLY_NAME,
+            ATTR_FRIENDLY_NAME: "Flick Power Price",
```

Comment:
This seems odd to set on an integration level?

Reply:
I thought so aswell, but didn't want to make it a breaking change. Remove it?

Reply:
Yes, let's remove it.